Cloud Application Manager/Core Concepts/architecture-components.md

```diff
@@ -28,33 +28,50 @@ This article is meant to assist users of Cloud Application Manager to learn abou
 
 * **CAM agents** - functions, interactions, installation, and requirements
 * **Websockets** - functions, interactions
-* API Services - functions, interactions
-* Orchestration Workflow - functions, interactions
-* Connectivity and Required Firewall Rules
+* **API Services** - functions, interactions
```

Comment:
Please, also update the metadata at the beginning of the file: date and authors at least